Spec/Auth.swift

```diff
@@ -1903,6 +1903,32 @@ class Auth : QuickSpec {
                         done()
                     }
                 }
+                
```

Comment:
I guess that previous developer just wanted to test that `issued` time is in 1 second range with server time, but I also cannot understand the pros of doing `time` function call to store server time.